src/sentry/db/router.py

```diff
@@ -0,0 +1,70 @@
+import django.apps
+
+DB_TABLE_TO_DB = {}
+
+
+def db_for_model(model_class):
+    if model_class._meta.app_label == "sentry":
+        return list(model_class._meta._ModelSiloLimit__silo_limit.modes)[0]._name_.lower() + "_silo"
```

Comment:
@RyanSkonnord Should we make the silo attributes public now that we have some more use-cases for them?

Reply:
I think this makes sense. Introspecting these is increasingly common.

Reply:
Yes, I'm coincidentally proposing something similar to solve a different problem in #38836.